File: kitten-house-care/src/main/java/org/jesperancinha/housing/configuration/CatCofiguration.java
package org.jesperancinha.housing.configuration;

import com.fasterxml.jackson.databind.ObjectMapper;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class CatCofiguration {
    @Bean
    public ObjectMapper objectMapper() {
        return new ObjectMapper();
    }
}

File: kitten-house-care/src/main/kotlin/org/jesperancinha/housing/configuration/CatCofiguration.kt
package org.jesperancinha.housing.configuration

import com.fasterxml.jackson.databind.ObjectMapper
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class CatCofiguration {
    @Bean
    fun objectMapper(): ObjectMapper {
        return ObjectMapper()
    }
}